<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Laguna Chat</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#titleLabel {
background: white;
color: blue;
font-size: 20px;
border: none;
border-bottom:  1px solid black;
padding: 5px;
}

#mainFrame {
border: none;
background: white;
}

#loginFrame {
background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #ddf, stop: 1 #aaf);
border: 1px solid gray;
padding: 10px;
border-radius: 25px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QFrame" name="mainFrame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="geometry">
        <rect>
         <x>32</x>
         <y>29</y>
         <width>721</width>
         <height>501</height>
        </rect>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="chatPage">
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="0">
          <widget class="QTextEdit" name="roomTextEdit">
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::TextBrowserInteraction</set>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLineEdit" name="sayLineEdit">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="maxLength">
            <number>255</number>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QPushButton" name="sayButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Say</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="2">
          <widget class="QListWidget" name="userListWidget"/>
         </item>
         <item row="0" column="0" colspan="3">
          <widget class="QLabel" name="titleLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Chatclient by Raphael Grewe</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="charLabel">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="loginPage">
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="1">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>100</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>223</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="1">
          <widget class="QFrame" name="loginFrame">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>0</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <property name="spacing">
             <number>20</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="labelUsername">
              <property name="text">
               <string>Email:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="LineEditEmail">
              <property name="text">
               <string/>
              </property>
              <property name="maxLength">
               <number>50</number>
              </property>
              <property name="placeholderText">
               <string>Address</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="labelPassword">
              <property name="text">
               <string>Password:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="LineEditPassword">
              <property name="text">
               <string/>
              </property>
              <property name="maxLength">
               <number>32</number>
              </property>
              <property name="echoMode">
               <enum>QLineEdit::Password</enum>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QPushButton" name="loginButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Login</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="2">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>223</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="1">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>267</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuProgram">
    <property name="title">
     <string>Program</string>
    </property>
    <addaction name="actionLogin"/>
    <addaction name="actionRegistration"/>
    <addaction name="actionConnect"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuExtras">
    <property name="title">
     <string>Extras</string>
    </property>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menuUser">
    <property name="title">
     <string>User</string>
    </property>
    <addaction name="actionAdd_User"/>
    <addaction name="actionDelete_User"/>
    <addaction name="separator"/>
    <addaction name="actionCreate_Room"/>
    <addaction name="actionDelete_Room"/>
   </widget>
   <addaction name="menuProgram"/>
   <addaction name="menuUser"/>
   <addaction name="menuExtras"/>
  </widget>
  <widget class="QStatusBar" name="statusBarMain"/>
  <action name="actionRegistration">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Registration</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionLogin">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionAdd_User">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Add User</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionDelete_User">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Delete User</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionCreate_Room">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Create Room</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionDelete_Room">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Delete Room</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>LineEditEmail</tabstop>
  <tabstop>LineEditPassword</tabstop>
  <tabstop>loginButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>sayLineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>sayButton</receiver>
   <slot>animateClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>331</x>
     <y>564</y>
    </hint>
    <hint type="destinationlabel">
     <x>768</x>
     <y>570</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>LineEditEmail</sender>
   <signal>returnPressed()</signal>
   <receiver>LineEditPassword</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>391</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>470</x>
     <y>238</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>LineEditPassword</sender>
   <signal>returnPressed()</signal>
   <receiver>loginButton</receiver>
   <slot>animateClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>417</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel">
     <x>389</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
